### PR TITLE
Translated alerting-policy-zh.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/project-user-guide/alerting/alerting-policy.md [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
 ---
 
-## Objective
+KubeSphere 支持针对节点和工作负载的告警策略。本教程演示如何为项目中的工作负载创建告警策略并配置邮件通知。有关如何为节点配置告警策略，请参阅[告警策略（节点级别）](../../../cluster-administration/cluster-wide-alerting-and-notification/alerting-policy/)。
 
-KubeSphere provides alert policies for nodes and workloads. This guide demonstrates how you, as a project member, can create alert policies for workloads in the project and configure mail notifications. See [Alerting Policy (Node Level)](../../../cluster-administration/alerting/alerting-policy/) to learn how to configure alert policies for nodes.
+## 准备工作
 
-## Prerequisites
+- 您需要用角色为 `platform-admin` 的帐户启用 [KubeSphere 告警和通知](../../../pluggable-components/alerting-notification/)。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/alerting/alerting-policy.md:14]: 您需要用角色为 `platform-admin` 的帐户启用 [KubeSphere 告警和通知](../../../pluggable-components/alerting-notification/)。
->
您已启用 [KubeSphere 告警和通知](../../../pluggable-components/alerting-notification/)。

File: content/zh/docs/project-user-guide/alerting/alerting-policy.md [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
+- 您需要用角色为 `platform-admin` 的帐户启用[邮件服务器](../../../cluster-administration/cluster-settings/mail-server/)。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/alerting/alerting-policy.md:15]: 您需要用角色为 `platform-admin` 的帐户启用[邮件服务器](../../../cluster-administration/cluster-settings/mail-server/)。
->
您已配置[邮件服务器](../../../cluster-administration/cluster-settings/mail-server/)。

File: content/zh/docs/project-user-guide/alerting/alerting-policy.md [hunk 3]
```diff
@@ -1,75 +1,75 @@
...
+您可以选择**部署**、**有状态副本集**和**守护进程集**三种工作负载作为监控目标。在本示例中，选择**部署**以及 `reviews-v1` 和 `details-v1` 作为监控目标，然后点击**下一步**。
 
-You can select three types of workloads as the monitoring targets: **Deployments**, **StatefulSets** and **DaemonSets**. Select **Deployments** as the type and `reviews-v1` and `details-v1` as monitoring targets, then click **Next**.
+![alerting_policy_workload_level_monitoring_target](/images/docs/zh-cn/alerting/alerting_policy_workload_level_monitoring_target.png)
 
-![alerting_policy_workload_level_monitoring_target](/images/docs/alerting/alerting_policy_workload_level_monitoring_target.png)
+### 步骤 4：添加告警规则
 
-### Task 4: Add Alerting Rules
+1. 点击**添加规则**创建告警规则。告警规则中包含指标类型、检查周期、连续次数、指标域值和告警级别等多个参数可供设置。
```
Comment (Sherlock113) [inline content/zh/docs/project-user-guide/alerting/alerting-policy.md:44]: 指标域值
->
指标阀值

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Thanks.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on comments.
/assign @Sherlock113

**Pull request merged**